addon/-private/environment/index.js

```diff
@@ -47,6 +47,7 @@ export default class Environment extends EmberObject {
   setUnknownProperty(key, value) {
     createEnvComputed(this, key, `__bound__.${findIndex(this.__bound__, key)}`);
     set(this, key, value);
+    this.trigger('change', { key, value });
```

Comment:
The `set` on the line above should already trigger this change, shouldn't it?

Reply:
it does- I'm surprised this didn't trigger the `onChange` action twice